From 0aef945671ed473a9e6b4931dad7945a2f924254 Mon Sep 17 00:00:00 2001
From: Vinzent Steinberg <Vinzent.Steinberg@gmail.com>
Date: Fri, 2 Aug 2019 17:25:22 +0200
Subject: [PATCH] Upgrade to Rand 0.7

This bumps the MSRV to 1.32 for the `rand` feature.

`SmallRng` is not used for the tests anymore, since enabling the corresponding
feature would force other crates to use the feature as well.
---
 Cargo.toml   | 2 +-
 src/crand.rs | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

Index: num-complex/Cargo.toml
===================================================================
--- num-complex.orig/Cargo.toml
+++ num-complex/Cargo.toml
@@ -32,7 +32,7 @@ features = ["i128"]
 default-features = false
 
 [dependencies.rand]
-version = "0.8"
+version = "0.7"
 optional = true
 default-features = false
 
